src/CustomPanel.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { DashboardPanelProps } from '@deephaven/dashboard';
+import Log from '@deephaven/log';
+import './CustomPanel.scss';
```

Comment:
Does this actually work? I had issues w/ it in another plugin and had to use [this](https://github.com/deephaven/deephaven-js-plugins/blob/main/plugins/dashboard-object-viewer/src/DashboardPlugin/DashboardPlugin.tsx#L9) instead

Reply:
I think it worked? Pretty sure I tested it with some odd styling...

Reply:
It indeed does not work. Trying with `?inline`

Reply:
Switched to `?inline`